javascript-light-bulb.tar/javascript-light-bulb/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Javascript Ligtbulb</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <script src="main.js"></script>
+  <div id="container">
+    <button id="button" onclick="changeStyle()">
```

Comment:
why are you using the `onclick` attribute on the button? It'd be better to use a querySelector in your javascript script file and add the `changeStyle` function you've added as your callback on the `addEventListener` method for the element after you've selected it.